packages/shared/identity-common/src/auth.ts

```diff
@@ -34,7 +35,23 @@ export const authenticatedInstanceFactory = (
         },
         ...getInstanceConfig(),
       });
+      axiosRetry(instance, {
+        retries: 3,
+        retryDelay: axiosRetry.exponentialDelay,
+        onRetry: (_, error) => {
+          console.warn(
+            'Error occurred during request, retrying',
+            error.message
+          );
+        },
+      });
```

Comment:
This is an attempt to deal with the socket hangup errors, which I suspect to be an intermittent network thing